apps/transport/lib/transport_web/controllers/aoms.ex

```diff
@@ -56,7 +56,36 @@ defmodule TransportWeb.AOMSController do
 
   @spec aoms :: [map()]
   def aoms do
-    datasets =
+    # Let’s fetch all GTFS datasets.
+    # This doesn’t include GTFS-Flex, although they are in public-transport and have a GTFS format on resource.
```

Comment:
```suggestion
    # This doesn’t include GTFS-Flex, although they are in public-transit and have a GTFS format on resource.
```

Déplacer ce commentaire sur la méthode ?